### ISSUE TITLE
StoriesTimeoutError in v3.1.8

### ISSUE THREAD

**Issue by dargmuesli** [reg-viz/storycap#608]
Following #594 / #605 the is no more error regarding the CSF file caching :+1: 

Instead, I now receive a `StoriesTimeoutError`: https://github.com/maevsi/maevsi/runs/6558288410?check_suite_focus=true#step:5:2211

```
Getting stories was failed. Make sure that your Storybook is rendered correctly.
```

**Comment by dargmuesli** [comment]
The issue is that in [stories-browser.ts#L73](https://github.com/mugi-uno/storycap/blob/f0dbba9af98beb116f91cea9619524b158dd2a37/packages/storycrawler/src/browser/stories-browser.ts#L73) `cacheAllCSFFiles` never returns in my project.
Thus not populating `cachedCSFFiles` and timing out in the configuring-wait-loop.

**Comment by carsugar** [comment]
I'm having the same issue.  @dargmuesli what did you do to resolve?

**Comment by dargmuesli** [comment]
I found no solution yet.

**Comment by LucaVazz** [comment]
From my testing this seems to be related to `storyStoreV7` - setting it to `false` in storybook's feature config enables storycap / storycrawler to run as expected.

**Comment by kodai3** [comment]
I'm having same issue, no workaround found yet ?

**Comment by vinhnt-github** [comment]
Hi @dargmuesli ,
Do you have the solution to fix this issue?
I’m having the same your issue.
Please help me.
Thanks so much!

**Comment by dargmuesli** [comment]
Please upvote the PR by giving a :+1: on it and do not add "me too" comments.
If there was a fix, this issue would link it and be cosed.

**Comment by sag1v** [comment]
> Please upvote the PR by giving a 👍 on it ..

@dargmuesli Which PR is addressing this issue?

**Comment by dargmuesli** [comment]
Sry, I meant "issue" not "pr"

**Comment by sag1v** [comment]
Yeah i don't think this issue is getting the attention it deserves. 
I tried to reproduce it with a "clean" project but i couldn't. It only happens when i migrate from react 16 to 18.
Latest storybook with react 16 works fine in my project. 
I would share the repo but unfortunately its private and i can't.

**Comment by sag1v** [comment]
Update - After some more playing around, i found out that it happens only when i enable some of my stories, meaning that when i only include some stories from my list (i have more than 500) it goes well, but when i render them all it crashes with the error above.
I didn't have time to check which stories are causing the issue exactly.
